### PR TITLE
added new messages in case importer

### PR DESCRIPTION
**Pull request by lwyszomi**
@snopoke 
http://manage.dimagi.com/default.asp?163003#1149109

### CODE REVIEW

File: corehq/apps/importer/const.py [hunk 1]
```diff
@@ -14,3 +14,4 @@ class ImportErrors:
     CaseGeneration = ugettext_noop('Case Generation Error')
     DuplicateLocationName = ugettext_noop('Duplicated Location Name')
     InvalidInteger = ugettext_noop('Invalid Integer')
+    ImportError = ugettext_noop('Import Error')
```
Comment (snopoke) [inline corehq/apps/importer/const.py]: rename to `ImportErrorMessage` to avoid shadowing the builtin name

**Comment by lwyszomi** [comment]
@snopoke 
Updated

File: corehq/apps/importer/tasks.py [hunk 2]
```diff
@@ -64,13 +64,22 @@ def do_import(spreadsheet_or_error, config, domain, task=None, chunksize=CASEBLO
...
-                user_id,
-            )
+            try:
+                submit_case_blocks(
+                    [ElementTree.tostring(cb.as_xml()) for cb in caseblocks],
+                    domain,
+                    username,
+                    user_id,
+                )
+            except Exception:
```
Comment (snopoke) [inline corehq/apps/importer/tasks.py]: There is also a class of errors that won't raise an exception but can be found by examining the form:
```python
response, xform, cases = submit_form_locally(...)
if not xform.is_error:
   problem = xform.problem
```

Sorry I should have mentioned that before

**Comment by lwyszomi** [comment]
@snopoke 
Updated

**Pull request merged**